### PR TITLE
Supports W8A8 quantization for more models

### PR DESCRIPTION
**Pull request by AllentDan**
Basically, all the AWQ models support W8A8 quantization too.

### CODE REVIEW

File: docs/zh_cn/supported_models/supported_models.md [hunk 1]
```diff
@@ -51,47 +51,47 @@ turbomind 引擎不支持 window attention。所以，对于应用了 window att
...
 |  InternLM2.5   |     7B      | LLM  |    Yes    |   Yes   |   Yes   | Yes  |  Yes  |
 |   Baichuan2    |     7B      | LLM  |    Yes    |   Yes   |   Yes   | Yes  |  No   |
 |   Baichuan2    |     13B     | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
 |    ChatGLM2    |     6B      | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
 |     Falcon     |  7B - 180B  | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
-|       YI       |  6B - 34B   | LLM  |    Yes    |   Yes   |   Yes   |  No  |  Yes  |
-|    Mistral     |     7B      | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
+|       YI       |  6B - 34B   | LLM  |    Yes    |   Yes   |   Yes   | Yes  |  Yes  |
+|    Mistral     |     7B      | LLM  |    Yes    |   Yes   |   Yes   | Yes  |  Yes  |
 |    Mixtral     | 8x7B, 8x22B | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
```
Comment (lvhan028) [inline docs/zh_cn/supported_models/supported_models.md:66]: Mixtral cannot?

Comment (AllentDan) [reply docs/zh_cn/supported_models/supported_models.md:66]: Pytorch engine did not support it yet.

File: docs/en/supported_models/supported_models.md [hunk 2]
```diff
@@ -51,47 +51,47 @@ The TurboMind engine doesn't support window attention. Therefore, for models tha
...
-|    Mistral     |     7B      | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
+|       YI       |  6B - 34B   | LLM  |    Yes    |   Yes   |   Yes   | Yes  |  Yes  |
+|    Mistral     |     7B      | LLM  |    Yes    |   Yes   |   Yes   | Yes  |  Yes  |
 |    Mixtral     | 8x7B, 8x22B | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
-|      QWen      | 1.8B - 72B  | LLM  |    Yes    |   Yes   |   Yes   |  No  |  Yes  |
-|    QWen1.5     | 0.5B - 110B | LLM  |    Yes    |   Yes   |   Yes   |  No  |  Yes  |
+|      QWen      | 1.8B - 72B  | LLM  |    Yes    |   Yes   |   Yes   | Yes  |  Yes  |
+|    QWen1.5     | 0.5B - 110B | LLM  |    Yes    |   Yes   |   Yes   | Yes  |  Yes  |
 |  QWen1.5-MoE   |    A2.7B    | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
-|     QWen2      | 0.5B - 72B  | LLM  |    Yes    |   Yes   |   No    |  No  |  Yes  |
```
Comment (zhulinJulia24) [inline docs/en/supported_models/supported_models.md:70]: where is qwen2.5

Comment (AllentDan) [reply docs/en/supported_models/supported_models.md:70]: Qwen2.5 shares the same structure with qwen2

Comment (lvhan028) [reply docs/en/supported_models/supported_models.md:70]: qwen2.5 is mentioned in #2849 @zhulinJulia24

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**